components/form/FormItem.razor.cs

```diff
@@ -223,15 +254,54 @@ void IFormItem.AddControl<TValue>(AntInputComponentBase<TValue> control)
                 builder.CloseComponent();
             };
 
-            if (control.ValueExpression is not null)
+			if (control.ValueExpression is not null)
                 _propertyReflector = PropertyReflector.Create(control.ValueExpression);
             else
                 _propertyReflector = PropertyReflector.Create(control.ValuesExpression);
 
-            if (_propertyReflector.RequiredAttribute != null)
+        }
+
+        ValidationResult[] IFormItem.ValidateField()
+        {
+            if (Rules == null)
             {
-                _labelCls = $"{_prefixCls}-required";
+                return Array.Empty<ValidationResult>();
             }
+
+            var results = new List<ValidationResult>();
+
+            var propertyInfo = _fieldIdentifier.Model.GetType().GetProperty(_fieldIdentifier.FieldName);
+            var displayName = string.IsNullOrEmpty(Label) ? _fieldIdentifier.FieldName : Label;
+
+            if (propertyInfo != null)
+            {
+                var validateMessages = Form.ValidateMessages ?? ConfigProvider?.Form?.ValidateMessages ?? new ValidateMessages();
+
+                foreach (var rule in Rules)
+                {
+                    var propertyValue = propertyInfo.GetValue(_fieldIdentifier.Model);
```

Comment:
If I understand this correctly, the value cannot really be different for each rule. If that is the case, then `propertyValue` could be obtained before `foreach` loop.

Reply:
Thank you for your review. All the situations you mentioned are correct. I have modified the code and adjusted some class names (using "Form" instead of "Rule").